### PR TITLE
[2.5] Drop Apache 2.2 support, update inline doc

### PR DESCRIPTION
**Pull request by andrerom**
As Apache 2.2 is not very relevant anymore, drop it before we release next LTS.

### CODE REVIEW

File: doc/apache2/Readme.md [hunk 1]
```diff
@@ -1,16 +1,20 @@
...
 
 
 Prerequisites
 -------------
 - Some general knowledge of how to install and configure Apache
-- Apache 2.x must be installed using [Apache MPM prefork](https://httpd.apache.org/docs/2.4/mod/prefork.html), *aka in "prefork mode"*.
+- Apache 2.4 must be installed using one of the following Multi-Processing Modules (MPM):
+    - [Event](https://httpd.apache.org/docs/2.4/mod/event.html), or alternatively [Worker](https://httpd.apache.org/docs/2.4/mod/worker.html), for use with `php-fpm` over FastCGI.
+    - [Prefork](https://httpd.apache.org/docs/2.4/mod/prefork.html), together with `mod_php`, running php as a Apache module.
+       - Useful for Backwards comparability or simple needs, for better performance _(less memory usage)_ pick a setup with `Event` MPM instead.
```
Comment (DominikaK) [inline doc/apache2/Readme.md]:
```suggestion
       - Useful for Backwards compatibility or simple needs. For better performance _(less memory usage)_ pick a setup with `Event` MPM instead.
```

File: doc/apache2/Readme.md [hunk 2]
```diff
@@ -157,4 +163,5 @@ following from the eZ installation root:
 
 The `NameVirtualHost` setting might already exist in the default configuration. Defining a new one will result in a
 conflict. If Apache reports errors such as `NameVirtualHost [IP_ADDRESS] has no VirtualHosts` or `Mixing * ports and
-non-* ports with a NameVirtualHost address is not supported`, try removing the `NameVirtualHost` line. For more details, see [NameVirtualHost directive](http://httpd.apache.org/docs/2.4/mod/core.html#namevirtualhost) section on the Apache documentation.
+non-* ports with a NameVirtualHost address is not supported`, try removing the `NameVirtualHost` line.
+For more details, see [NameVirtualHost directive](http://httpd.apache.org/docs/2.4/mod/core.html#namevirtualhost) section on the Apache documentation.
```
Comment (DominikaK) [inline doc/apache2/Readme.md]:
```suggestion
For more details, see [NameVirtualHost directive](http://httpd.apache.org/docs/2.4/mod/core.html#namevirtualhost) section in Apache documentation.
```

File: doc/apache2/Readme.md [hunk 3]
```diff
@@ -1,16 +1,20 @@
-Apache 2.2 / 2.4  configuration
-===============================
+Apache 2.4 configuration
+========================
 
-For recommended versions of [Apache](https://httpd.apache.org/), see [online eZ requirements](https://doc.ez.no/display/TECHDOC/Requirements).
+For recommended versions of [Apache](https://httpd.apache.org/), see [online eZ requirements](https://doc.ezplatform.com/en/2.5/getting_started/requirements/).
```
Comment (DominikaK) [inline doc/apache2/Readme.md]: Pro forma I'll mention that the link won't work until we have a 2.5 branch in the doc.

File: doc/apache2/Readme.md [hunk 4]
```diff
@@ -1,16 +1,20 @@
...
 
 
 Prerequisites
 -------------
 - Some general knowledge of how to install and configure Apache
-- Apache 2.x must be installed using [Apache MPM prefork](https://httpd.apache.org/docs/2.4/mod/prefork.html), *aka in "prefork mode"*.
+- Apache 2.4 must be installed using one of the following Multi-Processing Modules (MPM):
+    - [Event](https://httpd.apache.org/docs/2.4/mod/event.html), or alternatively [Worker](https://httpd.apache.org/docs/2.4/mod/worker.html), for use with `php-fpm` over FastCGI.
+    - [Prefork](https://httpd.apache.org/docs/2.4/mod/prefork.html), together with `mod_php`, running php as a Apache module.
```
Comment (DominikaK) [inline doc/apache2/Readme.md]:
```suggestion
    - [Prefork](https://httpd.apache.org/docs/2.4/mod/prefork.html), together with `mod_php`, running PHP as an Apache module.
```

File: doc/apache2/Readme.md (hunk 3 above)
Comment (andrerom) [reply doc/apache2/Readme.md]: On this wanted to ask if we can setup a hidden alias already (for latests for now) so we can start to prepare links to the 2019lts somehow

Comment (andrerom) [reply doc/apache2/Readme.md]: ok, lets do that

File: doc/apache2/Readme.md [hunk 5]
```diff
@@ -19,6 +23,8 @@ These examples are simplified to get you up and running, see [Virtual host templ
 
 #### Virtual Host
 
+_If you are new to Apache, consider checking tutorials online [like this one](https://websiteforstudents.com/how-to-run-apache2-with-php7-2-fpm-on-ubuntu-16-04-17-10-18-04/) first._
```
Comment (lserwatka) [inline doc/apache2/Readme.md]: I think we should remove this part. It does not look pro.

**Comment by andrerom** [comment]
updated @lserwatka

**Pull request merged**